File: Q10.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    float salario_fixo;
    float carros_vendidos;
    float valor_carro;
    float valor_das_vendas;
    float salario_final;
    cout << "Digite o salário fixo do vendedor: ";
    cin >> salario_fixo;
    cout << "Digite o número de carros vendidos pelo vendedor: ";
    cin >> carros_vendidos;
    cout << "Digite o valor fixo ganho por cada carro vendido: ";
    cin >> valor_carro;
    cout << "Digite o valor total das vendas do vendedor: ";
    cin >> valor_das_vendas;
    salario_final = salario_fixo + carros_vendidos * valor_carro + 3 * valor_das_vendas / 100;
    cout << fixed << setprecision(2);
    cout << "O salário final do vendedor será R$" << salario_final << "." << endl;
    return 0;
}

File: Q10.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class Q10 
{
    public static Scanner leia = new Scanner(System.in);

    public static void main(String args[]) 
    {
        int carros_vendidos;
        float salario_fixo, valor_carro, valor_das_vendas, salario_resultado;
        String salario_final;
        System.out.print("Digite o salário fixo do vendedor: ");
        salario_fixo = leia.nextFloat();
        System.out.print("Digite o número de carros vendidos pelo vendedor: ");
        carros_vendidos = leia.nextInt();
        System.out.print("Digite o valor fixo ganho por cada carro vendido: ");
        valor_carro = leia.nextFloat();
        System.out.print("Digite o valor total das vendas do vendedor: ");
        valor_das_vendas = leia.nextFloat();
        salario_resultado = salario_fixo + carros_vendidos * valor_carro + 3 * valor_das_vendas / 100;
        salario_final = new DecimalFormat("#.00").format(salario_resultado);
        System.out.println("O salário final do vendedor será R$" + salario_final + ".");
    }
}
